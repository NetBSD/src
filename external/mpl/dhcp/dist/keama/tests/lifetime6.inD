# DHCPv4 use ISC DHCP default lifetimes but not when configured.

min-lease-time 3600;
default-lease-time 7200;
max-lease-time 14400;
