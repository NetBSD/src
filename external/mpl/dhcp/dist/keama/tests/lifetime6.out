{
  # DHCPv4 use ISC DHCP default lifetimes but not when configured.
  "Dhcp6": {
    "min-valid-lifetime": 3600,
    "valid-lifetime": 7200,
    "max-valid-lifetime": 14400
  }
}
